recipes/console_bridge/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "console_bridge" %}
+{% set version = "0.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ros/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 9f024a38f0947ed9fa67f58829980c2d90d84740e6de20d75cb00866f07a7a0b
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+
+test:
+  requires:
+    - pkg-config
+  commands:
+    - pkg-config --modversion {{ name }}
+    - ls $PREFIX/lib/*{{ name }}*
+
+about:
+  home: http://github.com/ros/console_bridge
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'A ROS-independent package for logging that seamlessly pipes into rosconsole/rosout for ROS-dependent packages.'
+  dev_url: https://github.com/ros/console_bridge
+
+extra:
+  recipe-maintainers:
+    - jcarpent
```

Comment:
Note: jcarpent agreed to be a maintainer in https://github.com/conda-forge/staged-recipes/pull/8910#issuecomment-514527711.